PostProcessing/Runtime/PostProcessRenderContext.cs

```diff
@@ -1,42 +1,48 @@
 namespace UnityEngine.Rendering.PostProcessing
 {
+#if UNITY_2017_2_OR_NEWER
+    using XRSettings = UnityEngine.XR.XRSettings;
+#elif UNITY_5_6_OR_NEWER
+    using XRSettings = UnityEngine.VR.VRSettings;
+#endif
+
     // Context object passed around all post-fx in a frame
     public sealed class PostProcessRenderContext
     {
         // -----------------------------------------------------------------------------------------
         // The following should be filled by the render pipeline
 
         // Camera currently rendering
-        private Camera m_camera;
+        Camera m_Camera;
         public Camera camera
         {
-            get
-            {
-                return this.m_camera;
-            }
-
+            get { return m_Camera; }
             set
             {
-                this.m_camera = value;
+                m_Camera = value;
 
-                if (XR.XRSettings.isDeviceActive)
+                if (XRSettings.isDeviceActive)
                 {
-                    RenderTextureDescriptor xrDesc = XR.XRSettings.eyeTextureDesc;
-                    m_width = xrDesc.width;
-                    m_height = xrDesc.height;
-
-                    m_xrSinglePass = (xrDesc.vrUsage == VRTextureUsage.TwoEyes);
+#if UNITY_2017_2_OR_NEWER
+                    RenderTextureDescriptor xrDesc = XRSettings.eyeTextureDesc;
+                    width = xrDesc.width;
+                    height = xrDesc.height;
+#else
+                    width = m_Camera.pixelWidth * 2;
```

Comment:
I'm not sure how to address this pre-eyeTextureDesc.  We can't programatically determine single-pass being active.  For now, I'm fine with leaving a comment.  Maybe for pre-2017.2, we can add that explicit option to the Layer (or wherever) that allows for explicit doubling of the width.

In any case, since Mult-Pass works with more of these effects in the earlier versions, it makes sense to leave this un-doubled.